main.py

```diff
@@ -0,0 +1,31 @@
+# main click program to use GiphyCLI
+import os
+import click
+from giphy_cli import GiphyCLI
+
+@click.group()
+def gif():
+    pass
+
+@gif.command()
+@click.option("--api-key", envvar="GIPHY_API_KEY", required=True, help="Giphy API key")
```

Comment:
Oh, that's cool. I didn't realize you could use `envvar` in click! Where did you learn that?